vividus-plugin-web-app/src/main/java/org/vividus/ui/web/variable/BrowserWindowWidthDynamicVariable.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.ui.web.variable;
+
+import org.vividus.selenium.IWebDriverProvider;
+import org.vividus.ui.variable.AbstractWebDriverDynamicVariable;
+
+public class BrowserWindowWidthDynamicVariable extends AbstractWebDriverDynamicVariable
```

Comment:
I propose to use a single variable, browser-window-size.

Usage examples:
${browser-window-size.width}
${browser-window-size.heigh}

cc @valfirst  @avinBar